src/test/ui/lifetimes/issue-77175.rs

```diff
@@ -0,0 +1,14 @@
+#[deny(single_use_lifetimes)]
+// edition:2018
+// check-pass
+
+// error: lifetime parameter `'a` only used once
```

Comment:
Change this to make it clear that this was the previously emitted error that *shouldn't* happen.

Reply:
Added more comprehensive comment